build/Makefile

```diff
@@ -90,7 +90,6 @@ $(PYTHON_DIR) :
 	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade twisted
 	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade pyyaml
 	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade google
-	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade protobuf==3.19.0
 	. $(abspath $(DSTDIR)/bin/activate) && pip install secp256k1==0.13.2
```

Comment:
Why do we need secp256k1 after Sawtooth removal?

Reply:
see (https://github.com/hyperledger-labs/private-data-objects/pull/380#discussion_r1015986200)